src/main/resources/uk/ac/ebi/embl/api/validation/FixerMessages.properties

```diff
@@ -113,4 +113,6 @@ fixNoStopCodonMake3Partial=No stop codon at the 3' end of the CDS feature transl
 fixDeleteTrailingBasesAfterStopCodon =Partial codon appeared after the stop codon has been trimmed.
 fixInternalStopCodonMakePseudo=The protein translation of the protein coding feature contains internal stop codons. \"pseudo\" qualifier has been added.
 fixNoStartCodonMake5Partial=The protein translation of the protein coding feature does not start with a methionine. 5' partial applied to the location.
-fixCodonStartNotOneMake5Partial=5' partial applied to the location as the protein coding feature with start codon {0} must be 5' partial.
\ No newline at end of file
+fixCodonStartNotOneMake5Partial=5' partial applied to the location as the protein coding feature with start codon {0} must be 5' partial.
+
+AsciiCharacterFix_1=Non-ascii characters fix: \"{0}\" -> \"{1}\".
```

Comment:
should be better , if it is more verbal Non-ascii characters fixed from \"{0}\" to  \"{1}\".
Or even better